reference-architecture/gce-cli/ansible/playbooks/openshift-minor-upgrade.yaml

```diff
@@ -27,4 +27,3 @@
     containerized: false
     openshift_hosted_metrics_storage_kind: dynamic
     openshift_master_metrics_public_url: 'https://metrics.{{ osm_default_subdomain }}/hawkular/metrics'
-    openshift_hosted_metrics_storage_volume_size: 20Gi
```

Comment:
@pschiffe good catch. The run time values defined should now be correct and allow for metrics to be deployed at a specific dize